src/components/adaptive-cards/Component/Component.jsx

```diff
@@ -118,6 +120,10 @@ export default function Component({data, className, style: styleProp}) {
         break;
       case acPropTypes.action:
       case acPropTypes.children:
+        childrenWithSpacers = value.flatMap((child) => [{type: 'Spacer', separator: child.separator, spacing: child.spacing || 'default'}, child]).slice(1);
```

Comment:
no need for `|| 'default'` here, handle it in the Spacer component.

You can also simplify a bit and send all child props to the Spacer:

```js
[{...child, type: 'Spacer'}, child]
```